mis_builder_budget_product/readme/newsfragments/32.feature

```diff
@@ -0,0 +1 @@
+Create a new module to add features to MIS Builder Budget. For now, it adds the product_id to the Budget items by account.
```

Comment:
this filename seems wrong. If you keep it, it should be renamed to "36" so that it will link this pr #36

Reply:
also the .gitignore file is useless. Maybe you meant ".gitkeep"?

Reply:
Hi @simahawk , because the [PR 32](https://github.com/OCA/mis-builder-contrib/pull/32) was merged with nobump, so the bot didn't [run towncrier](https://github.com/OCA/oca-github-bot/blob/master/src/oca_github_bot/tasks/merge_bot.py#L147) last time